spec.go

```diff
@@ -6,4 +6,9 @@ type Logger interface {
 	// Log takes a sequence of alternating key/value pairs which are used to
 	// create the log message structure.
 	Log(v ...interface{}) error
+
+	// With returns a new contextual logger with keyvals appended to those
+	// passed to calls to Log. If logger is also a contextual logger
+	// created by With keyvals is appended to the existing context.
```

Comment:
@rossf7 I think it should rather be (missing comma) `If logger is also a contextual logger created by With, keyvals is appended to the existing context.` This is the `keyvals` which is appended.